flight/Modules/System/systemmod.c

```diff
@@ -124,6 +130,29 @@ int32_t SystemModInitialize(void)
 	SystemStatsInitialize();
 	FlightStatusInitialize();
 	ObjectPersistenceInitialize();
+
+	uint8_t platformType;
+	SystemSettingsPlatformTypeGet(&platformType);
+
+	switch (platformType){
```

Comment:
This is a crucial part. It only loads the relevant UAVO. This is to save memory on constrained targets. 

The downside is that it requires rebooting the vehicle when changing airframes. Not the end of the world, but it'd be nice to be able to have all these UAVOs in memory on a normal basis.